defaults/main.yml

```diff
@@ -57,6 +57,7 @@ etcd_settings:
   "logger": "zap" # Specify ‘zap’ for structured logging or ‘capnslog’.
   "log-outputs": "systemd/journal" # Specify 'stdout' or 'stderr' to skip journald logging even when running under systemd
   "enable-v2": "true" # enable v2 API to stay compatible with previous etcd 3.3.x (needed for flannel e.g.)
+  "discovery-srv": false # Replace with discovery domain to enable DNS SRV discovery
```

Comment:
What's the expected value for this variable? an string with a domain name or an YAML boolean? the comment and the default value seem contradictory.

Reply:
It's a domain name. You're right it's better to default to an empty string, since the truth check will be the same. Fixing.